05_Day_Arrays/05_day_starter/scripts/main.js

```diff
@@ -1,3 +1,83 @@
+//1.
+const emptyArray = [];
+
+//2.
+const favNumbers = [2, 6, 8, 67, 44];
+
+//3.
+favNumbers.length;
+
+//4.
+console.log(favNumbers[0]);
+console.log(favNumbers[2]);
+console.log(favNumbers[5]);
+
+//5.
+const mixedDataTypes = [
+    48, 
+    "Hello", 
+    false, 
+    {name: "Nicole", age: "34"}, 
+    null];
+
+console.log(mixedDataTypes);
+console.log(mixedDataTypes.length);
+
+//6.
+const itCompanies = ["Facebook", "Google", "Microsoft", "Apple", "IBM", "Oracle", "Amazon"];
+
+//7.
+console.log(itCompanies);
+
+//8.
+console.log(itCompanies.length);
+
+//9.
+console.log(itCompanies[0]);
+console.log(itCompanies[3]);
+console.log(itCompanies[6]);
+
+//10. 
+//11. 
+itCompanies[0] = "FACEBOOK";
+itCompanies[1] = "GOOGLE";
+itCompanies[2] = "MICROSOFT";
+itCompanies[3] = "APPLE";
+itCompanies[4] = "IBM";
+itCompanies[5] = "ORACLE";
+itCompanies[6] = "AMAZON";
+
+console.log(itCompanies);
+
+//12.
+itCompanies.toString();
+
+//15.
+itCompanies.sort();
+
+//16.
+itCompanies.reverse();
+
+//17.  console prints that 
+itCompanies.slice(0, 3);
+
+//18.
+itCompanies.slice(4, 7);
+
+//19.
+itCompanies.slice(3, 4);
+
+//20. 
+itCompanies.shift();
+
+//21.
```

Comment:
> Remove the middle IT company or companies from the array

Hint here is company or companies: you should remove one or two companies depending on if the value of `itCompanies.length` is odd or even.